src/elm/Auth/Action.elm

```diff
@@ -25,6 +25,8 @@ import Route.Path
 import View exposing (View)
 
 
+{-| Action : a type that represents the possible actions that can be taken in the application.
```

Comment:
the comments in this file will be overwritten by `elm-land generate`, unfortunately

Reply:
removed comments from this file.